src/plasma/plasma_manager.cc

```diff
@@ -235,12 +235,11 @@ struct PlasmaManagerState {
   /** The time (in milliseconds since the Unix epoch) when the most recent
    *  heartbeat was sent. */
   int64_t previous_heartbeat_time;
-  /** An object's object_id is added to this set immediately after
-   *  it is received and sealed. The object is removed in
-   *  process_add_object_notification, which is triggered by
-   *  the corresponding notification from the plasma store. */
-  // TODO(hme): change semantics to "received some bytes of object"
-  std::unordered_set<ObjectID, UniqueIDHasher> received_objects;
+  /** An ObjectID is added to this set if a shared buffer is
```

Comment:
Can you add something like "This is the subset of the ObjectIDs currently being transferred to this manager" As well as "Note that there is no guarantee that an ObjectID will appear in this set even if it is currently being transferred to this manager (because if a single transfer fails, we flush the entire set)"